.gitignore

```diff
@@ -69,3 +69,4 @@ docs/_build
 
 .direnv
 .envrc
+flake.lock
```

Comment:
The answer is possible `yes` in the future.
`flake.lock` help others reproduce the developing environment of the project.
However, the flake is the unstable feature of Nix, and keep changing now.
If one does not keep use nix from now, it might be useless in the future.